must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -145,6 +136,12 @@ echo "collecting dump of oc get pvc all namespaces" | tee -a  "${BASE_COLLECTION
 { oc get pvc --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/pvc_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect pvc --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
 
+# For volumesnapshot of all namespaces
+echo "collecting dump of oc get volumesnapshot all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get volumesnapshot --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/get_volumesnapshot_all_namespaces"
```

Comment:
add oc get -oyaml output also?

Reply:
AFAIK, that should come with the `inspect` command. @yati1998 can confirm.

Reply:
okay in that case we are good :+1:

Reply:
yes, it will come from oc inspect